addons/l10n_it_edi_withholding/models/account_edi_format.py

```diff
@@ -75,18 +75,20 @@ def _l10n_it_edi_get_extra_info(self, company, document_type, body_tree):
     def _import_fattura_pa_line(self, element, invoice_line_form, extra_info):
         messages_to_log = super()._import_fattura_pa_line(element, invoice_line_form, extra_info)
 
+        company = self.env["res.company"].browse([invoice_line_form['company_id']])
+        currency = self.env['res.currency'].browse([invoice_line_form['currency_id']])
+
         for withholding_tax in extra_info.get('withholding_taxes', []):
-            withholding_tags = element.xpath("Ritenuta")
+            withholding_tags = element.xpath('Ritenuta')
             if withholding_tags and withholding_tags[0].text == 'SI':
-                invoice_line_form.tax_ids |= withholding_tax
+                invoice_line_form["tax_ids"].append(fields.Command.link(withholding_tax.id))
```

Comment:
We could import Command to avoid fields.Command

Reply:
Done. 👍🏻